src/pages/dmt5/components/_side-tab.tsx

```diff
@@ -110,25 +110,6 @@ const TabList = styled.div<ContentProps>`
         margin: 0;
     }
 `
-
-const Desktop = styled(Show.Desktop)`
-    flex: 1;
-    width: 100%;
-`
-
-const Mobile = styled(Show.Mobile)`
-    @media ${device.tabletS} {
-        margin-bottom: 2.3rem;
-
-        &:last-child {
-            margin-bottom: 0;
-        }
-    }
-    @media ${device.mobileL} {
-        margin-bottom: 0;
-    }
-`
```

Comment:
Does this not remove the styles that are tied to those original components as well?

Reply:
yes , I commented it out initially to check if the design is being effected for device dimensions of tabletS and mobileS but did not see a difference. Hence I removed it and directly updated the Mobile and Desktop components.

Reply:
Thanks for the check 👍